### PR TITLE
Group by container and service

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

Our CPU metric panel always reported wrong results, it showed results like 4 CPU are used for a Zeebe container, where actually the limits are at ~1. This PR resolves this issue. See relates [slack thread](https://camunda.slack.com/archives/C037RS2JHB8/p1688641310918929)

It looks like in our current set up we scrape the metrics twice which results in wrong results. Furthermore, metrics can be reported twice for empty container and actual container.
service

We exclude the empty container name, and group now by service, container, and pod to see correct results.

Elasticsearch seems to contain two containers in our SaaS environment which is the reason why we want to see the container here as well.

container

We need to investigate where the second metric comes from, and who scrapes it. I was not able to pinpoint this yet, but in order to resolve the CPU issue for now I made the changes and the output a bit more verbose.

### CODE REVIEW

File: monitor/grafana/zeebe.json [hunk 1]
```diff
@@ -8749,7 +8749,7 @@
             "alertThreshold": true
           },
           "percentage": false,
-          "pluginVersion": "9.5.2",
+          "pluginVersion": "9.3.6",
```
Comment (npepinpe) [inline monitor/grafana/zeebe.json:8752]: 💭 Weird, all other olugins were upgrade but this one downgraded? 🤷‍♂️

**Review by npepinpe — APPROVED** [review]
Thanks for the quick fix! Would this apply to other container_ metrics?

File: monitor/grafana/zeebe.json (hunk 1 above)
Comment (Zelldon) [reply monitor/grafana/zeebe.json:8752]: Sorry I did a git add -p and probably added this wrongly.

**Pull request merged**